target/linux/ramips/dts/mt7621_mikrotik_routerboard-760igs.dts

```diff
@@ -51,6 +52,9 @@
 	label = "sfp";
 	phy-mode = "rgmii-rxid";
 	phy-handle = <&ephy7>;
+
+	nvmem-cells = <&macaddr_hard 0>;
```

Comment:
```suggestion
	nvmem-cells = <&macaddr_hard 5>;
```

Mikrotik devices seem to generally assign one MAC address per port/interface.
On this device in OEM, SFP gets last address. (In 7xx dtsi,) MAC address increments for each port.

Reply:
Interesting. This was not done.

Reply:
Done.